text/0030-protocol-messages.md

```diff
@@ -0,0 +1,246 @@
++ Feature name: `protocol-messages`
++ Start date: 2020-04-27
++ RFC PR: [iotaledger/bee-rfcs#30](https://github.com/iotaledger/bee-rfcs/pull/30)
++ Bee issue: [iotaledger/bee#70](https://github.com/iotaledger/bee/issues/70)
+
+# Summary
+
+This RFC introduces the IOTA protocol messages that were initially added in
+[IRI#1393](https://github.com/iotaledger/iri/pull/1393).
+
+# Motivation
+
+To be able to take part in the IOTA networks, Bee nodes need to implement the exact same protocol presented in this RFC
```

Comment:
I think we should make it clear that IRI gets replaced and we focus on being compatible with Hornet. Maybe replace the Motivation part with something like this:

> To be able to take part in the IOTA networks, IOTA node softwares need to implement the exact same protocol. Since IRI is being abandoned and replaced by Hornet and Bee, Bee does not aim for compatibility with IRI. This RFC therefore addresses the protocol changes introduced by Hornet.

Reply:
I feel like this is already addressed in the Drawbacks section. Maybe it's not clear enough and I can improve but I'm not sure the Motivation is the best place to say that.

Reply:
Yes understand. The Drawback sections notes that Bee will not be compatible with IRI, but it could be interpreted wrong when reading the Motivation part.
> [...] Bee nodes need to implement the exact same protocol presented in this RFC and currently being used by IRI nodes and HORNET nodes

Reply:
It's the same protocol but not the same version though 🤔

Reply:
Yes it's hard to get this right. But with the above mentioned part (https://github.com/iotaledger/bee-rfcs/pull/30#discussion_r423692148) IMO we don't have this problem and it is relatively easy to understand. But your right, not sure if it's the best place to say that.

Reply:
Addressed